CHANGELOG.md

```diff
@@ -8,6 +8,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ### Added
+Added 'MAPL_find_bounds => fidn_bounds' and 'MAPL_Interval => Interval' to MAPL.F90 for use when doing component level OpenMP
```

Comment:
You have a minor typo: fidn_bounds should be changed to find_bounds

Reply:
I pushed in a fix. Resolved!